common/security-features/subresource/image.py

```diff
@@ -79,7 +79,7 @@ def encode_string_as_bmp_image(string_data):
     img.putdata(color_data)
 
     # Flush image to string.
-    f = StringIO.StringIO()
+    f = six.StringIO()
```

Comment:
Generally it should be possible to use `io.BytesIO` or `io.StringIO` instead on both Python 2 and 3